clang/lib/Serialization/ASTWriterDecl.cpp

```diff
@@ -1259,6 +1259,7 @@ void ASTDeclWriter::VisitVarDecl(VarDecl *D) {
     VarDeclBits.addBit(D->isConstexpr());
     VarDeclBits.addBit(D->isInitCapture());
     VarDeclBits.addBit(D->isPreviousDeclInSameBlockScope());
+    VarDeclBits.addBit(D->hasInitWithSideEffects());
```

Comment:
I think you need to update the fixed width here https://github.com/llvm/llvm-project/blob/4084ffcf1e69b962e864aa138bb54dabbcec912f/clang/lib/Serialization/ASTWriterDecl.cpp#L2743.  That's likely what's causing the CI test failures.

Reply:
Thanks!